optimum/exporters/onnx/model_patcher.py

```diff
@@ -448,3 +449,45 @@ def __init__(
             self._patch_func = _prepare_decoder_attention_mask
             self._orig_func_name = "_prepare_decoder_attention_mask"
             self._orig_func = self._model_to_patch._prepare_decoder_attention_mask
+
+
+def is_open_clip_available():
+    return importlib.util.find_spec("open_clip") is not None
```

Comment:
Could you move this in `optimum/utils/import_utils.py`?

Reply:
Done.